lib/bamboo_cache.rb

```diff
@@ -0,0 +1,29 @@
+class BambooCache
+  def whos_out
+    @whos_out_data
+  end
+
+  def employees
+    @employees_data
+  end
+
+  def store_whos_out(data)
+    @whos_out_last_cache = Time.now.day
+    @whos_out_data = data
+  end
+
+  def store_employees(data)
+    @employees_last_cache = Time.now.day
+    @employees_data = data
+  end
+
+  def whos_out_needs_cache?
+    return true if Time.now.day != @whos_out_last_cache
+    false
+  end
+
+  def employees_needs_cache?
+    return true if Time.now.day != @employees_last_cache
```

Comment:
Same thing, you don't need to return true or false just `Time.now.day != @employees_last_cache`